packages/genesys-spark-components/src/components/beta/gux-side-panel/tests/gux-side-panel.spec.ts

```diff
@@ -0,0 +1,283 @@
+import { newSpecPage } from '@test/specTestUtils';
+import { MockHTMLElement } from '@stencil/core/mock-doc';
+import { GuxSidePanel } from '../gux-side-panel';
+import { GuxSidePanelHeading } from '../components/gux-side-panel-heading/gux-side-panel-heading';
+import { GuxModalSidePanel } from '../components/gux-modal-side-panel/gux-modal-side-panel';
+import { GuxDismissButton } from '../../../stable/gux-dismiss-button/gux-dismiss-button';
```

Comment:
Is adding GuxDismissButton to the list of components for each test necessary? I can't see any test that depends on it.

Reply:
Yes, no need to test the innards here.